### PR TITLE
add preflight-specific labels to the resulting container image

### PR DESCRIPTION
**Pull request by komish**
We pass through UBI's labels, so I figured we should add some of our own.

Signed-off-by: Jose R. Gonzalez <jose@flutes.dev>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,9 +17,23 @@ RUN make build RELEASE_TAG=${release_tag}
...
+ARG release_tag
+ARG preflight_commit
 ARG ARCH
 ARG OS
 
+# Metadata
+LABEL name="Preflight" \
+      vendor="Red Hat, Inc." \
+      maintainer="Red Hat OpenShift Ecosystem" \
+      version="1" \
```
Comment (acornett21) [inline Dockerfile:29]: Non-Blocking: What's the difference between `version` and `release_tag` in context of `metadata`? ie when should version here change?

**Review by acornett21 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (bcrochet) [reply Dockerfile:29]: If I'm seeing this correctly, I would think when the Dockerfile changes. But @komish would be better to answer this.

File: Makefile [hunk 2]
```diff
@@ -38,7 +38,7 @@ tidy:
 
 .PHONY: image-build
 image-build:
-	$(IMAGE_BUILDER) build -t $(IMAGE_REPO)/preflight:$(VERSION) .
+	$(IMAGE_BUILDER) build --build-arg release_tag=$(RELEASE_TAG) --build-arg preflight_commit=$(VERSION) -t $(IMAGE_REPO)/preflight:$(VERSION) .
```
Comment (bcrochet) [inline Makefile:41]: Non-blocking: I'd like to see these maybe passed in as a make envvar? Then they can be modified, added to, overridden from elsewhere (or even not present in a local build).

**Review by bcrochet — APPROVED** [review]
/lgtm

Just a comment for thought.

File: Dockerfile (hunk 1 above)
Comment (acornett21) [reply Dockerfile:29]: That makes sense to me.

**Pull request merged**